packages/toolbox/src/commands/chainlink/contractUtils.ts

```diff
@@ -36,8 +47,11 @@ export const runUpdate = async <T extends 'updateAggregator' | 'updateAggregator
       process.exit();
     }
 
-    const tx = await contract.functions[method](...neverParams);
+    const tx = await contract.functions[method](...neverParams, txOverrides);
     console.log(`Transaction: ${tx.hash}`);
+    await tx.wait();
+    console.log('Confirmed!');
+    console.log();
```

Comment:
delete this log?

Reply:
it's on purpose, to clearly separate multiple transactions if there are many

Reply:
Ha ok!
I am more used to do "\n"